packages/pdf/src/RPA/PDF/keywords/document.py

```diff
@@ -180,20 +198,16 @@ def template_html_to_pdf(
 
         .. code-block:: robotframework
 
-            *** Settings ***
-            Library    RPA.PDF
-
-            *** Variables ***
-            ${TEMPLATE}    order.template
-            ${PDF}         result.pdf
-            &{DATA}        name=Robot Generated
-            ...            email=robot@domain.com
-            ...            zip=00100
-            ...            items=Item 1, Item 2
-
-            *** Tasks ***
+            *** Keywords ***
             Create PDF from HTML template
-                Template HTML to PDF   ${TEMPLATE}  ${PDF}  ${DATA}
+                ${TEMPLATE}=    Set Variable    order.template
+                ${PDF}=         Set Variable    result.pdf
+                &{DATA}=        Create Dictionary
+                ...             name=Robot Generated
+                ...             email=robot@domain.com
+                ...             zip=00100
+                ...             items=Item 1, Item 2
+                Template HTML to PDF   template=${TEMPLATE}  output_path=${PDF}  variables=${DATA}
```

Comment:
You can wrap this line as:
```suggestion
                Template HTML to PDF   template=${TEMPLATE}
                ...    output_path=${PDF}  variables=${DATA}
```
so you make sure you won't get past the 88 chars limit during linting. (wasn't this already discovered on your side by `poetry run inv lint` ?)

Reply:
interestingly the lint didn't catch it

Reply:
Updated